elasticreindexer/process/reindexer_test.go

```diff
@@ -60,7 +60,7 @@ func testCopyMappingNoOverwriteShouldCreate(t *testing.T) {
 
 	r, _ := newReindexer(sourceClient, destinationClient, []string{"index"})
 
-	err := r.copyMappingIfNecessary(testIndex, false)
+	err := r.copyMappingIfNecessary(testIndex, false, false)
```

Comment:
declare constants for overwrite and skipMappings and give them as parameters for copyMappingIfNecessary